apps/web/app/(app)/onboarding/OnboardingEmailAssistant.tsx

```
@@ -0,0 +1,69 @@
+"use client";
+
+import { Suspense } from "react";
+import { useForm, SubmitHandler } from "react-hook-form";
```

Biome

[error] Some named imports are only used as types.

This import is only used as a type.

Importing the types with import type ensures that they are removed by the transpilers and avoids loading unnecessary modules.
Safe fix: Use import type.


(lint/style/useImportType)